typing_extensions/src_py3/typing_extensions.py

```diff
@@ -1755,3 +1772,165 @@ def get_type_hints(obj, globalns=None, localns=None, include_extras=False):
         if include_extras:
             return hint
         return {k: _strip_annotations(t) for k, t in hint.items()}
+
+elif HAVE_ANNOTATED:
+
+    # Prior to python 3.7 types did not have `copy_with`. A lot of the equality
+    # checks, argument expansion etc.. are done on the _subs_tree.
+    #
+    # >> U = Union[Optional[float], int]
+    # >> U1 = Union[T, int]
+    # >> U2 = U1[Optional[float]]
+    # >> U == U2
+    # True
+    # >> U2.__args__, U.__args__
+    # (typing.Union[float, NoneType],) (<class 'float'>, <class 'NoneType'>, <class 'int'>)
```

Comment:
I would remove these examples, they are not really helpful. Maybe instead mention which docstrings in `typing` to read?